pysph/sph/equation.py

```diff
@@ -549,7 +565,7 @@ def __init__(self, equations, real=True, update_nnps=False, iterate=False,
     # Non-public interface.
     ##########################################################################
     def __repr__(self):
-        cls = self.__class__.__name__
+        cls = self.name
```

Comment:
No, this should not be changed as a repr should also show how one can instantiate the group. Leave the `cls`, just make sure that name=xxx shows in the constructor arguments to the Group.

Reply:
Thanks, fixed.